wiki/People/BAT/en.md

```diff
@@ -0,0 +1,86 @@
+[BlueChinchompa](https://osu.titanic.sh/u/40)   |                     | Bosnian, Serbian, Croatian                  | Since the beginning
+[wrn81056](https://osu.titanic.sh/u/645)        |                     | French, Malaysian, Russian, German, Chinese | May 19th, 2024[^1] 
+[Digitalfear117](https://osu.titanic.sh/u/809)  | Beatmap File Expert |                                             | May 29th, 2024
+[Kurboh](https://osu.titanic.sh/u/810)          |                     | Croatian                                    | May 22nd, 2024[^3]
+[Nikku](https://osu.titanic.sh/u/811)           | Storyboarding       |                                             | May 19th, 2024[^1]
+[-Ady](https://osu.titanic.sh/u/821)            |                     |                                             | May 19th, 2024[^1]
+[Oposh](https://osu.titanic.sh/u/829)           |                     | Polish                                      | May 19th, 2024[^1]
+[Cartman](https://osu.titanic.sh/u/857)         |                     | Tagalog                                     | May 20th, 2024[^2]
+[iSlodinx](https://osu.titanic.sh/u/869)        |                     | Serbian, Croatian, Bosnian, some Macedonian | May 29th, 2024
+[arutama](https://osu.titanic.sh/u/905)         |                     | Indonesian                                  | June 14th, 2024[^4] 
+[Ignacio](https://osu.titanic.sh/u/943)         |                     | Spanish                                     | July 15th, 2024[^5]
+
+
+### Taiko Team Members
+
+User | Specialization | Additional Languages Spoken | Added Date 
+---|:---:|:---:|---
+[StreeX](https://osu.titanic.sh/u/67)           |   |          | May 19th, 2024[^1]
+[Cartman](https://osu.titanic.sh/u/857)         |   | Tagalog  | May 20th, 2024[^2]
+
+
+### Catch the Beat Team Members  
+User | Specialization | Additional Languages Spoken | Added Date 
+---|:---:|:---:|---
+[Digitalfear117](https://osu.titanic.sh/u/809)  | .osu mapping  |        | May 29th, 2024
+[Nikku](https://osu.titanic.sh/u/811)           | Storyboarding |        | May 19th, 2024[^1]
+[Oposh](https://osu.titanic.sh/u/829)           |               | Polish | May 19th, 2024[^1]
+
+
+### Mania Team Members
+User | Specialization | Additional Languages Spoken | Added Date 
+---|:---:|:---:|---
+[BlueChinchompa](https://osu.titanic.sh/u/40)  |   |   | Since the beginning 
+
+
+## Past Members
+
+User | Time in BAT
+---|:---:
+[Levi](https://osu.titanic.sh/u/2)   | Beginning -> May 19th, 2024
+[Spare](https://osu.titanic.sh/u/92) | December 17th, 2023 -> May 15th, 2024
+[Champ](https://osu.titanic.sh/u/96) | December 17th, 2023 -> May 15th, 2024
+[Fluxie](https://osu.titanic.sh/u/517)[^6] | May 19th, 2024 -> June  6th, 2024
+
+
+## History
+
+[Levi](https://osu.titanic.sh/u/2) Removed himself from BAT as soon as new members showed up to focus more on developing Titanic.
+[Fluxie](https://osu.titanic.sh/u/517) Got annoyed from being pinged and left.
```

Comment:
This shouldn't be mentioned whatsoever, sounds funny to me ngl, could be changed to something else to fit the reason.

Reply:
the Levi one I understand being in the history however the other two/three I don't really but they wanted it so they added it

Reply:
> This shouldn't be mentioned whatsoever, sounds funny to me ngl, could be changed to something else to fit the reason.

I personally think it should be mentioned.

Reply:
> the Levi one I understand being in the history however the other two/three I don't really but they wanted it so they added it

Are you talking about Spare & EZChamp? If so then I can somewhat agree with that.

Reply:
What I meant by "the other two/three" was fluxie, spare, and ezchamp. I personally don't think any of those 3 should be mentioned but it's up to you guys

Reply:
I talk about fluxie though.